terraform/core/storage.tf

```diff
@@ -41,6 +41,14 @@ resource "google_storage_bucket" "fmltc-gae-source" {
   depends_on = [google_project_service.gcp_services]
 }
 
+resource "google_storage_bucket_object" "with_support_from_google_cloud" {
+  name         = "static/WithSupportFromGoogleCloud.png"
+  source       = "${path.root}/../../server/static/WithSupportFromGoogleCloud.png"
+  content_type = "image/png"
+  bucket       = google_storage_bucket.fmltc.name
+  depends_on   = [google_storage_default_object_access_control.public_rule]
+}
+
```

Comment:
A bucket just for the image?  I'm not sure I understand the motivation for this change.

Reply:
It's a bucket object, right? Not a whole separate bucket.
I might not be doing this correctly. It's supposed to be in the same cloud storage bucket as the css/styles.css file (see the lines below this).

The motivation was that it is faster to redeploy static files (using deploy_static.sh) than it is to redeploy to app engine (using deploy_gae.sh) and I wanted faster turn-around time. Also, no need to make FIRST pay for the app engine request to retrieve this image.